docs/api/modal.md

```diff
@@ -39,17 +39,19 @@ A Modal is a dialog that appears on top of the app's content, and must be dismis
 
 When using `ion-modal` with Angular, React, or Vue, the component you pass in will be destroyed when the modal is dismissed. As this functionality is provided by the JavaScript framework, using `ion-modal` without a JavaScript framework will not destroy the component you passed in. If this is a needed functionality, we recommend using the `modalController` instead.
 
-It is recommended to using triggers to open inline modals. A trigger is an element reference, that when selected will automatically open a modal. Triggers are useful when presenting a modal from a button or similar interaction.
+import InlineModalTriggerExample from '@site/static/usage/modal/inline/basic/index.md';
 
-TODO: Playground Example
+<InlineModalTriggerExample />
```

Comment:
In the Stackblitz for React, I get an error after clicking the Cancel button (`modal.dismiss is not a function`).